native_words.asm

```diff
@@ -3089,21 +3089,10 @@ do_common:
 
                 ; now we compile six dummy bytes that LOOP/+LOOP will
                 ; replace by the actual LDA/PHA instructions
-                lda #5                  ; we don't really care about the value,
-                tay                     ; so we use 5 to be tricky
-_loop:
-                sta (CP),y
-                dey
-                bpl _loop
 
-                ; update CP
-                ina             ; we used 5 as a dummy value, this is why
-                clc
-                adc CP
-                sta CP
-                bcc +
-                inc CP+1
-+
+                ldy #6
+                jsr cmpl_inline        ; write 6 dummy bytes to runtime
```

Comment:
we actually don't need the write part at all, just the cp increment, but we have this routine already